### DISCUSSION TITLE
The issue is related to @types/node package

### DISCUSSION THREAD

**Discussion by Mohamed-Farag** [DefinitelyTyped/DefinitelyTyped#68359] [Request a new @types package]
Hello,

There is an issue regarding @types/node version "@types/node": "^20.10.5",

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/33036643/09b8db8f-9dc3-4b47-9694-dc65505e9496)

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/33036643/fc564c09-fecc-4291-8ff6-5ee622cb365b)

Could you please help in this?

Thanks

**Comment by ghost** [comment]
Fixed in PR: #68353
Try using `@types/node` version `20.11.9`
